torch/_dynamo/variables/tensor.py

```diff
@@ -883,15 +955,30 @@ def wrap(i, sub_proxy):
         return VariableTracker.build(tx, out)
 
     def method_backward(self, *args, **kwargs):
-        unimplemented("Tensor.backward")
+        unimplemented_v2(
+            gb_type="Unsupported Tensor.backward() call",
+            context=f"call_method {self} backward {args} {kwargs}",
+            explanation="Dynamo currently does not support tracing `Tensor.backward()`.",
+            hints=[*graph_break_hints.SUPPORTABLE],
```

Comment:
This is a fundamental graph break.

Reply:
Many thanks for your review. Have updated in this commit: https://github.com/pytorch/pytorch/pull/153146/commits/4d3fa9d50a33f27cc1b1839a4a0744a9809fc9f3